plugins/homepage/web_client/views/FrontPageView.js

```diff
@@ -10,7 +10,27 @@ wrap(FrontPageView, 'render', function (render) {
         method: 'GET',
         url: 'homepage/markdown'
     }).done(_.bind(function (resp) {
-        this.$el.html(renderMarkdown(resp['homepage.markdown']));
+        if (resp['homepage.markdown'] === '') {
+            restRequest({
+                method: 'GET',
+                url: 'homepage/settings'
+            }).done(_.bind(function (res) {
+                _.bind(render, this)();
+                console.log(this.$('.g-frontpage-subtitle').text());
+                console.log(res['homepage.header']);
+                if (res['homepage.header'] !== '') {
+                    this.$('.g-frontpage-title').text(res['homepage.header']);
+                }
+                if (res['homepage.subheading_text'] !== '') {
+                    this.$('.g-frontpage-subtitle').text(res['homepage.subheading_text']);
+                }
+                if (res['homepage.welcome_text'] !== '') {
+                    this.$('.g-frontpage-welcome-text').text(res['homepage.welcome_text']);
+                }
+            }, this));
+        } else {
+            this.$el.html(renderMarkdown(resp['homepage.markdown']));
+        }
     }, this));
```

Comment:
This is good for now, but we probably need a way to explicitly select whether the user wishes to completely replace the content via the markdown box, or change a few items via the other settings. Forcing the user to delete information (from the markdown box) just to activate a different config path is a UI failing in my opinion.

Reply:
Doing that, for now, force the markdown to be priority on the other settings, but I'm agree it shouldn't be the correct method to do that

Reply:
Ok, we can file an issue so we remember to address this later.